assets/prometheus-k8s/rules.yaml

```diff
@@ -92,6 +92,10 @@ spec:
     - expr: |
         sum(rate(container_cpu_usage_seconds_total{job="kubelet", metrics_path="/metrics/cadvisor", image!="", container!="POD"}[5m])) by (namespace)
       record: namespace:container_cpu_usage_seconds_total:sum_rate
+    - expr: |
+        kube_pod_container_status_last_terminated_reason{reason="OOMKilled",job="kube-state-metrics"}
+        * on (cluster, namespace, pod) group_left(node) max by(cluster, namespace, pod, node) (kube_pod_info)
+      record: kube_pod_container_status_last_terminated_reason:oom_total:sum
```

Comment:
Now there is no `sum` in the recording expression :thinking:

Reply:
> which gives you a split on namespaces and 39 new metrics into telemetry, or

That is most likely too many metrics, you get 10 series budget per component roughly, unless you get special approval from @smarterclayton and a valid reason why this is necessary.

Reply:
Thank you for the explanations. I changed it to the single metric so we can keep an eye on OOMs across the cluster.